### DISCUSSION TITLE
Scheduled workflows do not working properly

### DISCUSSION THREAD

**Discussion by HudsonLSchumaker** [community/community#27612] [Actions]
Hello,
I would like to know, why is not possible to have assurance on scheduled jobs when using self hosted runners?
I was searching here in the discussions and seems this is an issue since 2020.
What can be done to workaround this or what is the solution ?

thanks in advance.

**Comment by airtower-luna** [comment]
I assume it depends on the load on the GitHub systems that manage the runs.

> What can be done to workaround this or what is the solution ?

If you have a self-hosted runner you already have a server of your own. You can use an actual [cron job](https://en.wikipedia.org/wiki/Cron) or anything similar (e.g. systemd timer unit) to run whatever you need to run at a precise time.

**Reply by HudsonLSchumaker** [reply]
> If you have a self-hosted runner you already have a server of your own. You can use an actual [cron job]
(https://en.wikipedia.org/wiki/Cron) or anything similar (e.g. systemd timer unit) to run whatever you need to run at a precise time.

This is not a good option or me, using GH actions will fit more in our development process. For example, we have a code that has to run in a repository, have the workflows... so can create PRs to reviewed, merged and after that will be executing according the schedule. No bare metal access and work necessary.
thanks any way for the suggestion.
